network/src/commonMain/kotlin/com/wire/kalium/network/api/user/login/SSOResponse.kt

```diff
@@ -0,0 +1,80 @@
+package com.wire.kalium.network.api.user.login
+
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SSOResponse(
+    @SerialName("uri")
+    val uri: String?,
+    @SerialName("xml")
+    val xml: AuthnRequestSSO?
+)
+
+@Serializable
+data class AuthnRequestSSO(
```

Comment:
Do we really need to parse this? I think we are just opening a URL in browser for the SSO login.

Have you read through the SSO use cases?
https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/18448638/Single+Sign-On+and+automated+provisioning

Reply:
I know we only need the URL but was not sure if the extra information are needed for anything else.
I will delete the extra data